frontend/packages/kubevirt-plugin/integration-tests/tests/mocks/mocks.ts

```diff
@@ -15,26 +15,27 @@ import {
   resolveStorageDataAttribute,
   deepFreeze,
 } from '../utils/utils';
-import { Flavor, ProvisionSource } from '../utils/constants/wizard';
+import { Flavor } from '../utils/constants/wizard';
 import {
   NIC_MODEL,
   NIC_TYPE,
   DISK_INTERFACE,
   DISK_SOURCE,
   DISK_DRIVE,
 } from '../utils/constants/vm';
+import { ProvisionSource } from '../utils/constants/enums/provisionSource';
 
 export const provisionSources = {
```

Comment:
source can be included in the ProvisionSource object

Reply:
please also change `provisionSourceInputs`

Reply:
what is wrong with `provisionSourceInputs` ?

Reply:
the keys are a string and should rather refer to the enum. Anyway it fails currently since the method is a description and not the right constant.